codes/classical/q-ary_digits/reed_solomon.yml

```diff
@@ -22,23 +22,43 @@ features:
   decoders:
     - 'Berlekamp-Welch decoder \cite{manual:{E. R. Berlekamp and L. Welch, Error Correction of Algebraic Block Codes. U.S.
 Patent, Number 4,633,470 1986.}}, assuming that \(t \geq (n+k)/2\).'
+    - 'Gao decoder using extended Euclidean algorithm \cite{manual:{S. Gao, "A new algorithm for decoding Reed-Solomon codes." Communications, information and network security. Springer, Boston, MA, 2003. 55-68.}}.'
+    - 'Sudan list decoding algorithm, up to \(1-\sqrt{2R}\) proportion of errors \cite{manual:{M. Sudan, “Decoding Reed-Solomon codes beyond the error-correction bound,” Journal of Complexity, vol. 13, no. 1, pp. 180–193, 1997.}}'
+    - 'Sudan–Guruswami list decoding algorithm, up to \(1-\sqrt{R}\) proportion of errors \cite{manual:{V. Guruswami and M. Sudan, “Improved decoding of Reed-Solomon and algebraic-geometric codes,” IEEE Transactions on Information Theory, vol. 45, pp. 1757–1767, 1999.}}'
```

Comment:
you can just cite doi for papers without the bibliography details: \cite{doi:10.1109/SFCS.1998.743426}. Please do through and change these.